ProcessLib/ComponentTransport/CreatePorousMediaProperties.cpp

```diff
@@ -0,0 +1,91 @@
+/**
+ * \file
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreatePorousMediaProperties.h"
+
+#include "BaseLib/reorderVector.h"
+
+#include "MaterialLib/PorousMedium/Permeability/createPermeabilityModel.h"
+#include "MaterialLib/PorousMedium/Porosity/createPorosityModel.h"
+#include "MaterialLib/PorousMedium/Storage/createStorageModel.h"
+
+#include "MeshLib/Mesh.h"
+
+namespace ProcessLib
+{
+namespace ComponentTransport
+{
+PorousMediaProperties createPorousMediaProperties(
```

Comment:
:+1: . This class can be general one for all fluid flow processes. After this PR merged, I  may refractory the other flow processes by using this class to reduce the source code  duplication  about porous medium among process classes.